src/main/scala/dpla/ingestion3/OaiRecordHarvesterMain.scala

```diff
@@ -42,27 +43,27 @@ object OaiRecordHarvesterMain {
   val logger = LogManager.getLogger(OaiRecordHarvesterMain.getClass)
 
   def main(args: Array[String]): Unit = {
+    val oaiConf = new OaiHarvesterConf(args.toSeq)
 
-    validateArgs(args)
-    println(schemaStr)
+    // These are the minimum required parameters for an OAI harvest
+    val outputFile = oaiConf.outputDir.apply()
```

Comment:
For these apply() calls, the shorthand is just (). so you could change:

`oaiConf.outputDir.apply()` 

to 

`oaiConf.outputDir`

I'm thinking that since the oaiConf fields are named pretty obviously, you could avoid creating these variables here and just sub in the call to oaiConfDir.someField().